io.go

```diff
@@ -5,15 +5,18 @@ import (
 	"io"
 )
 
+// type ContextCloser has a Close method with context.
```

Comment:
just `ContextCloser is a Closer with a context argument`

Reply:
okie changed